fdbclient/NativeAPI.actor.cpp

```diff
@@ -2192,6 +2217,134 @@ ACTOR Future<Void> watchValue(Future<Version> version, Key key, Optional<Value>
 	}
 }
 
+ACTOR Future<Void> watchStorageServerResp(Key key, Database cx) {
+	loop {
+		try {
+			state Reference<WatchMetadata> metadata = cx->getWatchMetadata(key);
+			if (!metadata.isValid()) {
+				return Void();
+			}
+
+			state Future<Version> watchFutureSS = watchValue(Future<Version>(metadata->version), key, metadata->value, cx, metadata->info, metadata->tags);
+			Version watchVersion = wait(watchFutureSS);
+
+			metadata = cx->getWatchMetadata(key);
+			if (!metadata.isValid()) return Void();
+			
+			if (watchVersion >= metadata->version) { // case 1: version_1 (SS) >= version_2 (map)
+				cx->deleteWatchMetadata(key);
```

Comment:
Add some `TEST(true)` macros in here and various other places. These are used to determine when a certain path has been executed, so for some of these rarer cases it will be useful to have stats on when they occur.